src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -1684,7 +1683,13 @@ public void updateService(final String serviceId, final Long version, final Serv
     if (criteria.serviceName() != null) {
       filters.put("name", Collections.singletonList(criteria.serviceName()));
     }
+    if (criteria.label() != null) {
+      filters.put("label", Collections.singletonList(
+          String.format("%s=%s", criteria.label().name(), criteria.label().value())
```

Comment:
Usually with the docker APIs, you can filter on labels as "name=value" but also just "name" with any value. Is that not the case with services?